### PR TITLE
added additional information for chrony

### PR DESCRIPTION
**Pull request by muellerbe**
some additional data to chrony

### CODE REVIEW

**Comment by berendt** [comment]
Please use https://github.com/osism/osism-ansible/blob/master/playbooks/generic-chrony-force-sync.yml.

**Comment by muellerbe** [comment]
added the link to section

File: source/operations/generic.rst [hunk 1]
```diff
@@ -27,6 +27,14 @@ Force NTP sync
    200 OK
    200 OK
 
+you can use osism custom playbook, at ``environments/custom/playbook-force-timesync.yml``, with https://github.com/osism/osism-ansible/blob/master/playbooks/generic-chrony-force-sync.yml
+
+and execute with command
+
+.. code-block:: console
+
+   osism-run custom force-timesync -l <inventoryname>
```
Comment (berendt) [inline source/operations/generic.rst:36]: Simply use `osism-generic chrony-force-sync -l testbed-node-0.osism.local`.

File: source/operations/generic.rst [hunk 2]
```diff
@@ -27,6 +27,14 @@ Force NTP sync
    200 OK
    200 OK
 
+you can use osism custom playbook, at ``environments/custom/playbook-force-timesync.yml``, with https://github.com/osism/osism-ansible/blob/master/playbooks/generic-chrony-force-sync.yml
```
Comment (berendt) [inline source/operations/generic.rst:30]: This is not necessary. I added this playbook to the osism-ansible image.

Comment (muellerbe) [reply source/operations/generic.rst:30]: ok, thanks for this information, I delete this PR and branch

**Pull request closed without merging**